Index: Components/Pages/Home.razor
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>@page \"/\"\n@code {\n    // Written by Omar Khan for CS4485.001, starting March 21, 2025.\n    // NetID: OLK200000\n\t// Login page for the application\n}\n<PageTitle>Home</PageTitle>\n\n<h1>Hello, world!</h1>\n\n<p>Welcome to your new app.</p>\n\n<form action=\"/Quiz\" method=\"get\">\n    <label for=\"utdId\">UTD ID:</label>\n    <input type=\"text\" id=\"utdId\" name=\"utdId\" maxlength=\"10\" required>\n    <br>\n    <label for=\"password\">Password:</label>\n    <input type=\"password\" id=\"password\" name=\"password\" required>\n    <br>\n    <button type=\"submit\">Submit</button>\n</form>\n\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Components/Pages/Home.razor b/Components/Pages/Home.razor
--- a/Components/Pages/Home.razor	(revision 6f774c054874ac596b5e9f62516661b3c0a4bac2)
+++ b/Components/Pages/Home.razor	(date 1743023247737)
@@ -21,3 +21,9 @@
 </form>
 
 
+<form action="/Teacher" method="get">
+    
+    
+</form>
+
+
Index: Components/Layout/NavMenu.razor
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><div class=\"top-row ps-3 navbar navbar-dark\">\n    <div class=\"container-fluid\">\n        <a class=\"navbar-brand\" href=\"\">BlazorApp1</a>\n    </div>\n</div>\n\n<input type=\"checkbox\" title=\"Navigation menu\" class=\"navbar-toggler\" />\n\n<div class=\"nav-scrollable\" onclick=\"document.querySelector('.navbar-toggler').click()\">\n    <nav class=\"flex-column\">\n        <div class=\"nav-item px-3\">\n            <NavLink class=\"nav-link\" href=\"\" Match=\"NavLinkMatch.All\">\n                <span class=\"bi bi-house-door-fill-nav-menu\" aria-hidden=\"true\"></span> Home\n            </NavLink>\n        </div>\n    </nav>\n</div>\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Components/Layout/NavMenu.razor b/Components/Layout/NavMenu.razor
--- a/Components/Layout/NavMenu.razor	(revision 6f774c054874ac596b5e9f62516661b3c0a4bac2)
+++ b/Components/Layout/NavMenu.razor	(date 1743023071259)
@@ -12,6 +12,7 @@
             <NavLink class="nav-link" href="" Match="NavLinkMatch.All">
                 <span class="bi bi-house-door-fill-nav-menu" aria-hidden="true"></span> Home
             </NavLink>
+            
         </div>
     </nav>
 </div>
Index: Components/Pages/Teacher.razor
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>@page \"/Teacher\"\n<h3>Teacher</h3>\n\n@code {\n    //written by conan\n    //3/25/2025\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Components/Pages/Teacher.razor b/Components/Pages/Teacher.razor
--- a/Components/Pages/Teacher.razor	(revision 6f774c054874ac596b5e9f62516661b3c0a4bac2)
+++ b/Components/Pages/Teacher.razor	(date 1743022501277)
@@ -4,4 +4,14 @@
 @code {
     //written by conan
     //3/25/2025
-}
\ No newline at end of file
+    
+    
+    
+}
+
+<PageTitle>Teacher Placeholder Page</PageTitle>
+
+<h1> Hello this is the teacher placeholder used for testing!</h1>
+
+<p> Welcome to the teacher page which you can make future changes to this page</p>
+
